common-rest/src/test/java/org/uniprot/api/rest/validation/ValidDownloadRequestTest.java

```diff
@@ -0,0 +1,64 @@
+package org.uniprot.api.rest.validation;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import org.hibernate.validator.internal.engine.constraintvalidation.ConstraintValidatorContextImpl;
+import org.junit.jupiter.api.Test;
+import org.uniprot.api.rest.request.FakeDownloadRequest;
+
+class ValidDownloadRequestTest {
+
+    @Test
+    void when_fields_passed_then_invalid_request() {
```

Comment:
isn't it when fields passed with an invalid (not accepted) format?

Reply:
yes, renaming the method name